authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java

```diff
@@ -959,6 +1004,21 @@ protected void removePolicyByMetadataObject(AuthorizationMetadataObject authzMet
     }
   }
 
+  protected String getConfValue(Map<String, String> conf, String key, String defaultValue) {
+    if (conf.containsKey(key)) {
+      return conf.get(key);
+    }
+    return defaultValue;
+  }
+
+  protected abstract String getServiceType();
+
+  protected abstract Map<String, String> getServiceConfigs(Map<String, String> config);
+
+  protected int getPrefixLength() {
+    return prefixLength;
```

Comment:
I think we can remove prefixLength variable.

```
protected int getPrefixLength() {
    String.format("%s.", rangerAuthorizationProperties.getPropertiesPrefix()).length();
}
```

Reply:
`rangerAuthorizationProperties` isn't member variable.